src/camera.rs

```diff
@@ -38,8 +41,17 @@ impl CharacterControllerCameraOf {
             enable_smoothing: true,
             step_smooth_time: Duration::from_millis(200),
             teleport_detection_distance: 10.0,
+            yank_speed: 210.0_f32.to_radians(),
         }
     }
+
+    /// Sets the yank speed.
+    ///
+    /// Expects `degrees_per_sec` (e.g., 180.0 for a half-turn per second).
+    pub fn with_yank_rate(mut self, degrees_per_sec: f32) -> Self {
+        self.yank_speed = degrees_per_sec.to_radians();
+        self
+    }
```

Comment:
mind removing this method? 
- IMO it's unexpected to have the builder for one property and not for others
- Everything in Bevy expects radians, so this using degrees is also unexpected

Reply:
Hm I guess default doesn't work since it has a relationship. I don't want to do this in `surf.rs`, but the `yank_speed` should be lower for surf than the default imo:

```rs
        .insert(CharacterControllerCameraOf {
            character_controller: player,
            enable_smoothing: true,
            step_smooth_time: Duration::from_millis(200),
            teleport_detection_distance: 10.0,
            yank_speed: 80.0_f32.to_radians(),
        });
```

Reply:
```rust
.insert(CharacterControllerCameraOf {
   yank_speed: 80.0_f32.to_radians(),
   ..CharacterControllerCameraOf::new(player)
})
```
should work I believe